app/bundles/FormBundle/Controller/Api/FormApiController.php

```diff
@@ -30,14 +44,6 @@ public function initialize(ControllerEvent $event)
         parent::initialize($event);
     }
 
-    /**
-     * {@inheritdoc}
-     */
-    protected function preSerializeEntity(&$entity, $action = 'view')
```

Comment:
Why was this removed? I worry about changing the behavior in this PR

Reply:
It's never used.